packages/@lwc/engine-server/src/apis/render-component.ts

```diff
@@ -5,8 +5,10 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
 import { createVM, connectRootElement, LightningElement } from '@lwc/engine-core';
+import type { RendererAPI } from '@lwc/engine-core/types/renderer';
 import { isString, isFunction, isObject, isNull } from '@lwc/shared';
 
+import * as renderer from '../renderer';
```

Comment:
import * is not good a determining types, probably another point to make the renderer an actual export of an object called renderer.

Reply:
Let's make sure to address this before merging.